kornia/augmentation/augmentation.py

```diff
@@ -563,7 +564,7 @@ class CenterCrop(AugmentationBase2D):
                   [-1.2633,  0.3500]]]])
     """
 
-    def __init__(self, size: Union[int, Tuple[int, int]], align_corners: bool = True,
+    def __init__(self, size: Union[int, Tuple[int, int]], align_corners: Optional[bool] = None,
```

Comment:
The default was `True` so setting it to `None` is a BC breaking change. Was this on purpose? `align_corners` is not mentioned in the docstring.